modules/API/pages/gsql-endpoints.adoc

```diff
@@ -3166,6 +3174,24 @@ Sample Response::
 {"error":false,"message":"","version":{"schema":1,"edition":"enterprise","api":"v2"},"results":[{"p1":["hello","world"]}]}
 ----
 --
+Sample Request For Interpreting Anonymous Query Using Request Header::
++
+--
+[source,bash]
+----
+curl -s -u tigergraph:tigergraph -H 'Workload-Queue: <Workload Queue Name>' 'localhost:14240/gsqlserver/interpreted_query' \
+-d 'INTERPRET QUERY () FOR GRAPH g { print 2; }'
+----
+
+Sample Request For Interpreting Anonymous Query Using GSQL Command::
++
+--
+[source,bash]
+----
+curl -s -H "Authorization: Bearer <USER TOKEN>" -H 'Workload-Queue: <Workload Queue Name>' 'localhost:14240/gsqlserver/interpreted_query' \
+-d 'INTERPRET QUERY -QUEUE OLTP () FOR GRAPH g { print 2; }'
```

Comment:
I said on Friday to make the example style similar to that of the existing examples.
I'm guessing you need a better understanding of HTTP in general and TigerGraph's REST servers in order to understand which parts you can change and which you can't. 

In brief, don't blindly copy `-s -H "Authorization: Bearer <USER TOKEN>"`. Check the other examples.
Don't blindly use `localhost:14240/gsqlserver/`. Check the other examples for the GSQL endpoints page.